apertium-eng.eng.rlx

```diff
@@ -329,6 +344,33 @@ SELECT V IF (-1 (prn pers)) ;
 SELECT SUB:1 Be IF (1 Ing) (0/* Have) ;
 SELECT Ger IF (-1/* Be) (1 To) ;
 
+SELECT Adv IF (0 Pr) (-1 Vblex) ;
+#Example (on):
+#He talked on.
+
+SELECT Cnjadv IF (0 Pr) (1 Det) (2 N) ;
+#Example (for):
+#Please get fish for the cat.
```

Comment:
In this sentence, that should be `pr` though, right?  You can't tell apart `cnjadv` and `pr` based on a following Det N anyway, though, since they can both be followed by that.

Reply:
Has this been addressed?

Reply:
Maybe this rule should just be removed?

Reply:
Probably.